Header header

# Current cartesian task state values
geometry_msgs/Pose current_pose
geometry_msgs/Twist current_velocity

#Desired cartesian task state values
geometry_msgs/Pose desired_pose
geometry_msgs/Twist desired_velocity
geometry_msgs/Accel desired_acceleration
